homeassistant/components/sensibo/climate.py

```diff
@@ -103,7 +125,7 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
         pysensibo.SensiboError,
     ) as err:
         _LOGGER.error("Failed to get devices from Sensibo servers")
-        raise PlatformNotReady from err
+        raise ConfigEntryNotReady from err
```

Comment:
This needs to be raised from '__init__.py'

Move the code that sets up the connection there and store it in hass.data[DOMAIN][entry.entry_id]